frontend/src/components/activity/ButtonNestedContentNodeAdd.vue

```diff
@@ -117,6 +121,9 @@ export default {
         contentType.name !== 'ResponsiveLayout' || this.parentContentNode.parent === null
       )
     },
+    showChecklistNode(contentType) {
+      return contentType.name === 'Checklist' ? this.featureChecklistEnabled : true
```

Comment:
i would have expected
```suggestion
      return this.featureChecklistEnabled && contentType.name === 'Checklist'
```

Reply:
```suggestion
      return contentType.name !== 'Checklist' || this.featureChecklistEnabled
```